tests/gdx-tests-iosrobovm/robovm.xml

```diff
@@ -5,6 +5,7 @@
   <arch>thumbv7</arch>
   <target>ios</target>
   <iosInfoPList>Info.plist.xml</iosInfoPList>
+  <treeShaker>conservative</treeShaker>
```

Comment:
Are you sure this is needed? As far as I know, the default setting is `conservative`.

Reply:
It's not "needed" here because IPA size is not that relevant for tests but I think it's good to have the same one that we have on `robovm.xml` gdx-setup. `conservatiove`setting doesn't harm and is totally save because it only prunes especifically anootated classes. Tree shaker on latest Mobivm is currently `none` by default iirc.